.github/workflows/check_issue_label.yml

```
@@ -0,0 +1,115 @@
+
+name: Check Issue Labels
+
+on:
+  issues:
+    types:
+      - labeled
+
+env:
+  PAT_TOKEN: ${{ secrets.TEAM_READ_PAT }}  # PAT with read:org scope
+
+permissions: {}
+
+jobs:
+  check-label:
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+
+    steps:
+      - name: Validate issue label
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ env.PAT_TOKEN }}
+          script: |
+            const { context } = require('@actions/github');
+            const octokit = require('@octokit/rest').Octokit.defaults({
+              auth: process.env.PAT_TOKEN
+            });
+            // ======================
+            // SECURE INPUT HANDLING
+            // ======================
+            const sanitizedLabel = (context.payload.label?.name || '').trim();
+            const actorUsername = (context.actor || '').trim().toLowerCase();
+            const issueNumber = Number(context.issue.number) || 0;
+            const { owner, repo } = context.repo;
+            // Validation checks
+            if (!/^[\w\s-:]+$/i.test(sanitizedLabel)) {
+              throw new Error(`Invalid label format: ${sanitizedLabel}`);
```

```suggestion
              throw new Error(`Invalid label: ${sanitizedLabel}`);
```

what you're printing in the error message is the label not the format